package.json

```diff
@@ -69,6 +69,6 @@
     "glob": "7.2.3",
     "husky": "7.0.4",
     "jest": "29.7.0",
-    "webpack-dev-server": "5.0.4"
+    "webpack-dev-server": "^4.15.1"
```

Comment:
we should match what frontend-build uses. Context https://github.com/edx/frontend-app-exams-dashboard/pull/29#discussion_r1698503705